src/icons/PlusFat.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
Could this be replaced by Add instead?

Reply:
Yes, I believe so. Removed plusfat in latest commit.